### DISCUSSION TITLE
Preexisting Pirate code site

### DISCUSSION THREAD

**Discussion by ParkerH27** [PSDTools/app#16] [General]
https://piratecode.psdr3.org/welcome

**Comment by ParkerH27** [comment]
Essentially now we need to find someone that worked on it last year and update what they did to make it waaaay better

**Reply by lishaduck** [reply]
Only Seniors would have, and we only know that @conn-lang worked on it.

**Reply by Conn-Lang** [reply]
Was there something you all needed help with?

**Reply by lishaduck** [reply]
No, sorry about that. There must have been a moth in GitHub's famously buggy notifications; it should [only notify people](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#mentioning-people-and-teams) if they're a member of the owning organization. I was using it as a fast way to link to your account for future reference. However, @ParkerH27 might have a different answer regarding if we needed anything. He's been puzzled by PowerSchool's lack of API documentation for their public-facing REST API. If you have anything to say, @PetalCat, you should speak up.
_**CC**: @ParkerH27, @PetalCat_
